### DISCUSSION TITLE
DFS2 to TIFF method

### DISCUSSION THREAD

**Discussion by miab-dhi** [DHI/mikeio#215] [Ideas]
Hi @ecomodeller and @jsmariegaard 

Is it possible to add a DFS2 method to convert DFS2 to TIFF and vice versa. I am aware there is a method of doing this ArcGIS if MIKE URBAN is installed. But having this functionality in mikeio would be great :).

Also, how can one efficiently search for available map projection strings? I created a DFS2 from a collection of TIFFs and wanted to project the data into "WGS 84 / UTM zone 18N".  After some digging in some other files I finally found the correct projection string.

-Michael

**Comment by ecomodeller** [comment]
Fastest way to find a WKT formatted projection string is probably to use the search function from [Spatial reference](https://spatialreference.org/) e.g. [https://spatialreference.org/ref/?search=utm+18n](https://spatialreference.org/ref/?search=utm+18n)

**Reply by miab-dhi** [reply]
Thanks. I also have noticed the spatial subpackage in mikeio which has a CRS class. There's a few methods in there that help generate the MIKE specific projection. Grateful this has been added to mikeio.
